### DISCUSSION TITLE
Why eglot--snippet-expansion-fn is ignore if company-tng-mode is enable

### DISCUSSION THREAD

**Discussion by tshu-w** [company-mode/company-mode#1313] [Q&A] [answered]
I'm coming from https://github.com/joaotavora/eglot/issues/884#issuecomment-1067698924 and I find if I enable company-tng-mode `eglot--snippet-expansion-fn` is ignored. However, I `(advice-remove #'eglot--snippet-expansion-fn #'ignore)` I find nothing wrong. Is there any reason to do this?

**Comment by tshu-w** [comment, marked as answer]
Sorry I find it https://github.com/company-mode/company-mode/issues/731#issuecomment-341151284

**Reply by tshu-w** [reply]
@dgutov I wander if there is anything update since I'm not facing a error if I `(advice-remove #'eglot--snippet-expansion-fn #'ignore)`

**Reply by dgutov** [reply]
It all depends on which completion backends you are using.

AFAIK both LSP clients have usability problems with `company-tng-mode` without this advice.
